fairlearn/metrics/_metric_frame.py

```diff
@@ -404,7 +445,7 @@ def group_max(self) -> Union[Any, pd.Series, pd.DataFrame]:
         else:
             return result
 
-    def group_min(self) -> Union[Any, pd.Series, pd.DataFrame]:
+    def group_min(self, errors: str = 'raise') -> Union[Any, pd.Series, pd.DataFrame]:
```

Comment:
We generally prefer to avoid repeating string literals like this - can you turn them into constants?

Reply:
Although I have many times advocated for this sort of change, I have recently learned that many think of this quite differently. I believe in scikit-learn they always duplicate it when it's in the signature. @adrinjalali may have thoughts.

Reply:
Not sure what the best way is to make this a constant. Should it be something like this?
```
_VALID_ERROR_STRINGS = [ 'raise', 'coerce' ]
_RAISE_ERROR_STRING = 'raise'
_COERCE_ERROR_STRING = 'coerce'
```

Reply:
yeah, I very much prefer having the actual strings here. It makes it much easier to read the code.

Reply:
You mean actual strings in the function signature @adrinjalali ?

For other uses @bram49 , I meant roughly that, except you'd define
```python
_VALID_ERROR_STRINGS = [ _RAISE_ERROR_STRING, _COERCE_ERROR_STRING ]
```

Reply:
I meant for almost everywhere. I find `'raise'` much more readable than `_RAISE_ERROR_STRING`

Reply:
@adrinjalali  has more experience of this sort of thing than I do; I'm happy to go with this suggestion.